src/components/PanelV2/PanelV2.js

```diff
@@ -102,7 +102,13 @@ export default class PanelV2 extends Component {
               <header ref={this.header} className={`${namespace}__header`}>
                 {title && (
                   <div className={`${namespace}__header__container--title`}>
-                    <div className={`${namespace}__header--title`}>{title}</div>
+                    {typeof title === 'string' ? (
+                      <h2 className={`${namespace}__header--title`}>{title}</h2>
```

Comment:
Decided to go with an `h2` instead of an `h1` here to mimic the structure of the `Modal`: https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/components/Modal/Modal.js#L343